src/app.js

```diff
@@ -1 +1,38 @@
 'use strict';
+/* eslint-disable no-console */
+
+const { numberToGuess } = require('./generateRandomNumber');
+const { isNumberValid } = require('./isNumberValid');
+const { countBullsAndCows } = require('./countBullsAndCows');
+const { terminal } = require('./terminal');
+
+let counter = 0;
+
+function play() {
+  terminal.question('Please, enter 4 digits.', (userGuess) => {
+    if (!isNumberValid(userGuess)) {
+      console.log('You should enter 4 different digits!');
+
+      play();
+
+      return;
+    }
+
+    counter++;
+
+    const { bulls, cows } = countBullsAndCows(userGuess, numberToGuess);
+
+    if (bulls === 4) {
+      console.log(`You won! You guessed the number
+        ${numberToGuess.join('')} from ${counter} tries!`);
+      terminal.close();
+
+      return;
+    }
```

Comment:
Would be better to do `if(userGuess === numberToGuess)` before using `countBullsAndCows` so you will not do useless operations